test/solveEquation/solveEquation.test.js

```diff
@@ -6,21 +6,21 @@ const solveEquation = require('../../lib/solveEquation');
 const NO_STEPS = 'no-steps';
 
 function testSolve(equationString, outputStr, debug=false) {
-  const steps = solveEquation(equationString, debug);
-  let lastStep;
-  if (steps.length === 0) {
-    lastStep = NO_STEPS;
-  }
-  else {
-    lastStep = steps[steps.length -1].newEquation.print();
-  }
```

Comment:
Same thing as above.  All code should go in and `it`... or `beforeEach` or `afterEach`.